App/App.js

```diff
@@ -14,7 +15,8 @@ export default function App() {
   } else {
     return (
       <SafeAreaProvider>
-        <Navigation colorScheme={colorScheme} />
+        {/*<Navigation colorScheme={colorScheme} />*/}
+        <CalendarScreen></CalendarScreen>
```

Comment:
Deze moet aangeroepen worden in navigation -> index.js.

moet je even een nieuwe route aanmaken en dan de CalendarScreen component meegeven.